data/attributes.yml

```diff
@@ -30209,6 +30209,36 @@ base_attributes:
   values:
   - vehicle_fluid_type__concentrate
   - vehicle_fluid_type__prediluted
+- id: 3415
+  name: Caffeine content
+  description: Indicates the amount of caffeine present in the coffee product, aiding consumers in choosing based on their desired caffeine intake level
```

Comment:
I would make this shorter and add an example:
"Indicates the amount of caffeine present in a product, e.g. decaffeinated, low caffeine"